packages/integration-tests/projects/suite-web/tests/settings/t1-device-settings.test.ts

```diff
@@ -2,6 +2,10 @@
 // @retry=2
 
 describe('T1 - Device settings', () => {
+    afterEach(() => {
+        cy.task('stopEmu');
+    });
+
     it('pin mismatch', () => {
         cy.task('startEmu', { version: Cypress.env('emuVersionT1'), wipe: true });
```

Comment:
Lines 10 and 11 are the same for both tests. I'd suggest removing them from both tests and adding them back to the `beforeEach` method.

Reply:
done, sent new PR, previous version was not created by me, i just added new test, now all test share `beforeEach` method